internal/worker/remoterelationconsumer/localunitrelations/package_test.go

```diff
@@ -2,3 +2,5 @@
 // Licensed under the AGPLv3, see LICENCE file for details.
 
 package localunitrelations
```

Comment:
You've renamed various members/methods/functions to be explicit about offerers and consumers instead of locals and remotes.

But here you still have a package sub-path `remoterelationconsumer/localunitrelations`